### ISSUE TITLE
Fix Request: false positive in AvoidThreadUnsafeJaxbUsage for non-jaxb Validator

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#169]
Your own written Validator should not raise a violation.

```
class Validation {
        Validator validator; // fine, self defined Validator from same package
}
```
